### DISCUSSION TITLE
All models disappeared after update

### DISCUSSION THREAD

**Discussion by j-taptok** [open-webui/open-webui#3414] [Q&A]
Hi,

After updating to the latest version all the connections were lost (Anthropic and Azure AI). I can't find where to reconfigure them. Does anyone know where can this be done?

**Comment by justinh-rahb** [comment]
Let me guess.. you went from 0.1.125 to latest? https://docs.openwebui.com/migration#migrating-from-internal-to-external-litellm

**Reply by j-taptok** [reply]
Hi @justinh-rahb seems like it. I see there is a need to setup another contianer for LiteLLM but how can I configure Azure AI and Anthropic if I already did the upgrade and lost the previous configuration? Is there a way to roll it back?
